suite-native/atoms/src/OrderedListItem.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactNode } from 'react';
+
+import { Color, TypographyStyle } from '@trezor/theme';
+
+import { Box } from './Box';
+import { Text } from './Text';
+
+type OrderedListItemProps = {
+    children: ReactNode;
+    order: number;
+    variant?: TypographyStyle;
+    color?: Color;
+};
+
+export const OrderedListItem = ({ children, order, variant, color }: OrderedListItemProps) => (
+    <Box flexDirection="row">
+        <Text variant={variant} color={color} style={{ width: 24, textAlign: 'center' }}>
```

Comment:
`width: 24` sounds very arbitrary here, what if order is `100`? I thinks it's okey solution for AboutViewOnlyBottomSheet, but then it doesn't belongs to universal atoms, what do you think?

Reply:
1. I have missed here that I haven't used applyStyle. I can fix that.

2. **24** is width taken from the figma. I think that if we run into situation where we have > 99, we will know that and the probability, that this will happen in really small. This nicely works for < 10, works okay for 10-99. We have more atoms that are not 100% bulletproof so I guess this is okay to have it here since it will probably works each time we use it.

Reply:
37b4ea1d71